Jenkinsfile

```diff
@@ -52,6 +53,9 @@ elifeLibrary {
     elifeMainlineOnly {
         stage 'Merge to master', {
             elifeGitMoveToBranch commit, 'master'
+            if (isNew) {
```

Comment:
also the order is probably better than tagging first and then merging, as a released tag should always come from `master`

Reply:
Yes, that was my thinking.